src/ducks/previousAnnotations.js

```diff
@@ -99,6 +99,7 @@ const constructAnnotations = (reductions, frame) => {
         const textOptions = constructText(annotation, i);
         const data = {
           points, frame, textOptions,
+          consensusReached: annotation.consensus_score >= 3,
```

Comment:
Recommendation: move the value `3` to the config, so we can adjust it more easily in the future.